qadi_hypothesis.py

```
@@ -0,0 +1,154 @@
+    )
+    parser.add_argument(
+        "--verbose", "-v",
+        action="store_true",
+        help="Show detailed evaluation scores"
+    )
+    
+    args = parser.parse_args()
+    
+    # Validate temperature if provided
+    if args.temperature is not None:
+        if args.temperature < 0.0 or args.temperature > 2.0:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)